gno.land/cmd/gnoweb/main.go

```diff
@@ -16,6 +16,30 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
+// Authorized external image host providers.
+var cspImgHost = []string{
+	// Gno-related hosts
+	"https://gnolang.github.io",
+	"https://assets.gnoteam.com",
+	"https://sa.gno.services",
+
+	// Other providers should respect DMCA guidelines.
```

Comment:
how can we have this comment and then have ipfs? lol

Reply:
i guess the hosts themselves do respect the guidelines... still it's funny.

Reply:
These gateways respect dmca requests